docs/devGuide/development/workflow.md

```diff
@@ -126,6 +126,22 @@ The sections below has more information about various stages of submitting a PR.
 
   </modal>
 
+#### Keeping your fork up to date
+
+When new PRs are merged into the `master` branch, your `master` branch will be out of sync with the main repository. One way to update your branches to branch from the latest master is as follows:
+
+1. Go to your fork on Github and click 'Sync Fork'
+
+1. Checkout your `master` branch and `git pull`
```

Comment:
```suggestion
1. Checkout and sync your local `master` (the name of your local branch that tracks the remote master) branch with `git checkout master && git pull`
```

Or suggest to have the commands in a separate code block, same for the following, for clarity.

Reply:
Sounds like a good change! I will add the commands in a separate code block for this and the merge/rebase as well!